feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go

```diff
@@ -214,7 +214,7 @@ func (tc *testCase) configureDUT(t *testing.T) {
 	tc.configDstAggregateDUT(agg, &dutDst)
 	aggPath := d.Interface(tc.aggID)
 	fptest.LogYgot(t, tc.aggID, aggPath, agg)
-	aggPath.Replace(t, agg)
+	aggPath.Update(t, agg)
```

Comment:
We definitely want to keep Replace.

The tc.setupAggregateAtomically() was guarded by a deviation because some vendors make it impossible to create an empty lag before adding members to it. It only contained enough update to allow the lag to exist.

The replace here should contain the actual config that the test plan intended.

Reply:
Thank you for the quick response, we'll take a look.

Reply:
Any update?

Reply:
We are working with the engineering team to resolve the issue.

Reply:
Hi @manan-patel , any new updates?

Reply:
WE have fixed this issue on our side. Please close the PR.